js/data.js

```diff
@@ -0,0 +1,92 @@
+import {getRandomPositiveInteger, getRandomPositiveFloat, getRandomArrayElement, getRandomArray} from './util.js';
+
+const ADS_NUMBERS = 10;
+
+const ADS_TYPES = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel'];
```

Comment:
```suggestion
  'hotel'
];
```
сделай красиво везде)